ui/narrative/methods/view_proteinstructures/spec.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name" : "View ProteinStructures",
+  "ver" : "1.0.0",
+  "authors" : [ ],
+  "contact" : "http://kbase.us/contact-us/",
+  "visble" : true,
```

Comment:
typo: `visible`

Reply:
This is an embarrassing error--obviously we have been copy/pasting from existing examples.  I found 53 of `visble` in the module.  So I decide to change all of them with a hope that my change won't cause any interruption.

Reply:
Wow! Yeah, that's a good find.